docker/1.4.1/py2/mms-entrypoint.py

```diff
@@ -0,0 +1,13 @@
+import shlex
```

Comment:
remove the old entrypoint script

Reply:
The old entrypoint also uses config.properties, but the python one doesn't. Should I also remove that?

Reply:
Sai and I looked at existing cpu and gpu docker files and found that they use the shell script. So we keep it for now until we update the cpu and gpu images for py2